data/publiccloud/terraform/ec2.tf

```diff
@@ -53,7 +53,7 @@ resource "aws_security_group" "basic_sg" {
         from_port   = 0
         to_port     = 0
         protocol    = "-1"
-        cidr_blocks = ["0.0.0.0/0"]
+        cidr_blocks = ["213.151.95.130/32", "195.135.220.0/22"]
```

Comment:
@cfconrad this is Czech NAT address and the German range. Is that OK for you?

Reply:
Yeah that is actually a good hint! We should add similar restrictions to other CSPs as well.
Also I think we only use SSH port. So we could limit it as well?

Reply:
@jlausuch @asmorodskyi wdyt?

Reply:
In QAM also other ports may be used. I would keep the source IP limitation.

Reply:
We used 0.0.0.0/0, which is the rule type `ALLOW ALL`  because we are not testing any security and did not care much about it since we remove the instances after the test...  I am ok with adding some extra security, but please make sure this new ingress rule works with our setup in OSD.

Reply:
@jlausuch thank you. I did so, the whole SUSE IP range is allowed here.